Library/Formula/chuck.rb

```diff
@@ -11,6 +11,9 @@ class Chuck < Formula
     sha256 "d5f7372ca9f939763a4ffd3894d8384797417cdb3455899f80da0f188c84f812" => :mountain_lion
   end
 
+  # Fixes OS X version detection to include 10.11 through (thinking ahead) 10.13
```

Comment:
Have not heard back yet. Their dev community doesn't seem to be very active (there was an 8 month span between Oct 2014 and June 2015 with zero mailing list activity), so it might be a bit before it's handled.

Looks like they use [a Wiki page](http://wiki.cs.princeton.edu/index.php/ChucK/Bugs/Reports) for bug tracking. Maybe I should just post it there and link to that?